src/App.css

```diff
@@ -44,7 +44,8 @@ ion-button {
 ion-button.main {
   --border-radius: 16px;
   --box-shadow: 0 2px 6px 0 rgb(0, 0, 0, 0.25);
-  width: 150px;
+  min-width: 150px;
```

Comment:
I guess you changed the width of the button because the word "mark" in Tamil is long. Is there another correct and short version to translate "Mark period"? If the word "அடையாளப்படுத்தவும்" is the most correct, then we'll leave it.

Reply:
Which is why I used that word, but now I have changed it from 'அடையாளப்படுத்தவும்' to 'குறி, also I remove Css which i added Before